app/2.0/toolbox/app-storage.md

```diff
@@ -4,7 +4,7 @@ title: App storage
 
 <!-- toc -->
 
-**The app storage elements are prerelease.** APIs may be subject to change.
+**The app storage elements are prerelease.** APIs may be subject to change. <--- Still prerelease???
```

Comment:
Nope https://github.com/PolymerElements/app-storage/releases 

Fixed in new PR